google_shopping_functions.js

```diff
@@ -5,26 +8,78 @@ var getItemsCount = function(itemData){
 	return itemData.items.length;
 };
 
-//
-//DEFINE FUNCTIONS HERE
-//
+// Question 1
+var getItems = function(objectData) {
+	return objectData.items;
+}
 
 
+console.log(getItems(data));
 
+//Question 2
 
+var items = getItems(data); 
+var getItemsbyBrand = function(items, brand) {
+	var brandItems = [];
+	for (var i = 0; i < data["items"].length; i++ ) {
```

Comment:
Here, instead of 
```
i < data["items"].length
```
You can just do
```
i < items.length
```
Because your function above returns data["items"]